attestation-service/docker/as-grpc/Dockerfile

```diff
@@ -17,26 +18,37 @@ RUN if [ "${ARCH}" = "x86_64" ]; then curl -L https://download.01.org/intel-sgx/
     apt-get update && apt-get install -y libsgx-dcap-quote-verify-dev; fi
 
 # Build and Install gRPC attestation-service
-RUN cargo install --path attestation-service --bin grpc-as --features grpc-bin --locked
+RUN cargo install --path attestation-service --bin grpc-as --features grpc-bin,${VERIFIER} --locked && \
+    strip /usr/local/cargo/bin/grpc-as
```

Comment:
why do we strip the binary?

Reply:
Yea to make the binary smaller as more layers are added to the dockerfile

Reply:
hmm, symbols are useful/required for debugging. by default, in an unconstrained environment I wouldn't strip release binaries. If we still want to do that, we can [put it in Cargo.toml](https://doc.rust-lang.org/cargo/reference/profiles.html#strip), in which we can pick more granular size optimizations and it's a bit less obscure.

But again, for a webservice, I wouldn't optimize for size without a good reason to do so.

Reply:
Ok. So let me delete the strip here and if needs we can optimize the size later